plugin.py

```diff
@@ -1,3 +1,4 @@
+from LSP.plugin.core.protocol import ExecuteCommandParams
```

Comment:
I know this is merged already but this is unused now, I guess. You can remove it at your discretion :)